airflow/cli/cli_parser.py

```diff
@@ -153,6 +153,12 @@ def __init__(self, flags=None, help=None, action=None, default=None, nargs=None,
     ),
     choices=tabulate_formats,
     default="fancy_grid")
+ARG_COLOR = Arg(
+    ('--color',),
+    help="do emit colored output (default: auto-detect)")
+ARG_NO_COLOR = Arg(
+    ('--no-color',),
```

Comment:
I think it's possible to do this in ArgParse without a separate option. One moment.

Reply:
Oh yes. `choices`. I will update PR.

Reply:
Nope, I'm making that up.